docker/nginx/conf.d/server/server.api

```diff
@@ -72,6 +72,29 @@ location /skynet/stats {
     proxy_pass http://sia:9980/skynet/stats;
 }
 
+location /nginx/stats {
```

Comment:
is this something we want to expose?

Reply:
idk, it feels like we should have an aggregate from all servers exposed and not just a single server data since that's really tied to our current load balancing architecture, same goes for /skynet/stats